src/components/classifier/SwipeClassifier.js

```diff
@@ -257,55 +240,30 @@ export class SwipeClassifier extends React.Component {
           { this.state.isQuestionVisible ? classification : tutorial }
           { this.state.isQuestionVisible ? unlinkedTask :null }
         </ClassificationPanel>
-        { this.state.isQuestionVisible ? needHelpText : null }
+        { this.state.isQuestionVisible && this.props.task.help ? <NeedHelpButton onPress={() => this.classifierContainer.displayHelpModal()} /> : null }
```

Comment:
Will this still evaluate to true if `this.props.task.help` is an empty object - `{}`? I think that is the default for a task, so you might want to double check this.

Reply:
good call. Will fix that